device/ospf/ospfv2/router.yaml

```diff
@@ -0,0 +1,171 @@
+components:
+  schemas:
+    Device.Ospfv2:
+      description: >-
+        A container of properties for an OSPFv2 router and its interfaces & Route Ranges.
+      type: object
+      required: [router_id, interfaces, name]
+      properties:
+        name:
+          x-include: ../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        router_id:
```

Comment:
Can we have a object with default choice of auto which should be defined to mean IPv4 address of first interface assigned to OSPFv2 router .  Idea is to be able to just add ospfv2 in a device and everything comes up with DUT with iface type as bcast  without needing any extra config.